### PR TITLE
r-aer: a new package.

### PR DESCRIPTION
**Pull request by weijianwen**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-aer/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    url      = "https://cran.r-project.org/src/contrib/AER_1.2-5.tar.gz"
+
+    version('1.2-5', '419df9dc8ee6e5edd79678fee06719ae')
+
+    depends_on('r-car', type=('build', 'run'))
+    depends_on('r-lmtest', type=('build', 'run'))
+    depends_on('r-sandwich', type=('build', 'run'))
+    depends_on('r-survival', type=('build', 'run'))
+    depends_on('r-zoo', type=('build', 'run'))
+    depends_on('r-formula', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-aer/package.py]: Can you add the dependency version requirements:
https://cran.r-project.org/web/packages/AER/index.html

File: var/spack/repos/builtin/packages/r-aer/package.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class RAer(RPackage):
+    """Functions, data sets, examples, demos, and vignettes
+    for the book Christian Kleiber and Achim Zeileis (2008),
+    Applied Econometrics with R, Springer-Verlag, New York.
+    ISBN 978-0-387-77316-2."""
+
+    homepage = "https://cran.r-project.org/web/packages/AER/index.html"
+    url      = "https://cran.r-project.org/src/contrib/AER_1.2-5.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-aer/package.py]: Needs a `list_url`:
```python
list_url = "https://cran.rstudio.com/src/contrib/Archive/AER"
```

Comment (weijianwen) [reply var/spack/repos/builtin/packages/r-aer/package.py]: Done.

File: var/spack/repos/builtin/packages/r-aer/package.py (hunk 1 above)
Comment (weijianwen) [reply var/spack/repos/builtin/packages/r-aer/package.py]: Done.

File: var/spack/repos/builtin/packages/r-aer/package.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    list_url = "https://cran.rstudio.com/src/contrib/Archive/AER"
+
+    version('1.2-5', '419df9dc8ee6e5edd79678fee06719ae')
+
+    depends_on('r@2.13.0:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-aer/package.py]: `type=('build', 'run')`

Comment (weijianwen) [reply var/spack/repos/builtin/packages/r-aer/package.py]: Fixed.

**Pull request closed without merging**